cypress/e2e/debug_download.cy.js

```diff
@@ -89,7 +89,7 @@ describe('Debug tab download', function () {
   // TODO: Fix issue and re-enable test.
   xit('Download all open reports', function () {
     const downloadsFolder = Cypress.config('downloadsFolder');
-    cy.get('[data-cy-debug="tableBody"]').find('tr').should('have.length', 2);
+    cy.get('[data-cy-debug="tableBody"]').get('tbody').find('tr').should('have.length', 2);
```

Comment:
Theres a lot of places where you check the length of the table, to follow the [DRY rule](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), make it a function:
```ts
Cypress.Commands.add('checkDebugTableLength', (n) => {
  if (n === 0){
    cy.get('[data-cy-debug="tableBody"] tbody').find('tr').should('not.exist');
  } else {
    cy.get('[data-cy-debug="tableBody"] tbody').find('tr').should('have.length', n);
  }
});

```

Reply:
added a global function that asserts the length of the tbody for n length now.